lib/generators/blacklight/assets/importmap_generator.rb

```diff
@@ -45,27 +40,38 @@ def append_blacklight_javascript
       end
 
       def add_stylesheet
-        if File.exist? 'app/assets/stylesheets/application.bootstrap.scss'
-          if ENV['CI']
-            run "yarn add file:#{Blacklight::Engine.root}"
-          else
-            run "yarn add blacklight-frontend@#{Blacklight::VERSION}"
-          end
+        return generate_with_sassc_rails unless used_bootstrap_css?
```

Comment:
I would find this more readable if it were

```
unless used_bootstrap_css?
   generate_with_sassc_rails
   return
```

(Unless you actually want to use the return value of `generate_with_sassc_rails`). 

This code is getting pretty convoluted and branching, keeping it readable seems good, what do you think?

Reply:
Sure, done.